src/gc.c

```diff
@@ -1196,6 +1196,16 @@ static NOINLINE jl_taggedvalue_t *add_page(jl_gc_pool_t *p) JL_NOTSAFEPOINT
     return fl;
 }
 
+// called into by LLVM-generated code
+JL_DLLEXPORT jl_value_t *jl_gc_pool_alloc_outer(jl_ptls_t ptls, int pool_offset,
+                                          int osize)
```

Comment:
My main comment is about trying to find a more informative name for this, like maybe:
- `jl_gc_pool_alloc_from_gen`
- `jl_gc_pool_alloc_from_llvm_gen`
- `jl_gc_pool_alloc_from_generated`...
- `jl_gc_pool_alloc_that_will_be_called_from_generated_code`... i dunno...

Blech naming is hard